tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -127,10 +127,13 @@ public function testListDatabases()
 
         $this->_sm->dropAndCreateDatabase('test_create_database');
         $databases = $this->_sm->listDatabases();
+        /** left for debug purposes */
+        $databases_ = [];
+        foreach ($databases as $database) {
+            $databases_[] = is_array($database) ? strtolower($database['name']) : $database;
+        }
 
-        $databases = array_map('strtolower', $databases);
```

Comment:
Please DO ADD a test to demonstrate your issue, rather than modifying existing tests.